operator-tests/src/test/resources/features/operator-deployment.feature

```diff
@@ -202,6 +202,28 @@ Feature: Operator Deployment
     Then wait for integration with name: "sql-sql-jaeger" to become active
     And check that jaeger pod "jaeger-all-in-one" is collecting metrics for integration "sql-sql-jaeger"
 
+  @ENTESB-15540
+  @operator-addons-jaeger-hybrid
+  @operator-addons
+  @operator-addons-jaeger
+  Scenario: Syndesis Operator - Addons - Jaeger - hybrid
+    When  deploy Syndesis CR from file "spec/addons/jaeger-hybrid.yml"
+    Then wait for Syndesis to become ready
+      And check that the pod "syndesis-jaeger" has not appeared
+    When create jaeger cr from "jaeger-syndesis-for-hybrid" file
+    Then wait for pod "syndesis-jaeger" is running
+      And check that the pod "syndesis-jaeger" has appeared
```

Comment:
probably these two should be switched, as when the pod is running, then it probably already appeared :grin: